pkg/backend/remote/remote.go

```diff
@@ -261,7 +261,7 @@ func (rf *Factory) Create(address string) (types.Backend, error) {
 	}
 
 	if replica.State != "closed" {
-		return nil, fmt.Errorf("Replica must be closed, Can not add in state: %s", replica.State)
+		return nil, fmt.Errorf("replica must be closed, Can not add in state: %s", replica.State)
```

Comment:
Need to update the assertion that is using this error message in the integration test. Similar below.
https://github.com/longhorn/longhorn-engine/blob/192e2a6922bfec7ca54ce8de67dc193cbec07c92/integration/core/test_cli.py#L1459